bohr/snorkel_utils.py

```diff
@@ -206,12 +206,7 @@ def issues(self) -> Issues:
                 df = df.loc[[self.sha]]
                 for sha, issue in df.iterrows():
                     labels = issue.labels
-                    if pd.isna(labels):
-                        labels = []
-                    elif isinstance(labels, float):
-                        labels = [str(labels)]
-                    else:
-                        labels = labels.split(', ')
+                    labels = list(filter(None, labels.split(', ')))
```

Comment:
what will happen if the label is a number e.g. '8.0'?

Reply:
OK, now it should work. Pandas is driving me crazy: I tested with integers, which it did not parse. But you just reminded me that I should check floats and indeed, it parsed them. So we force 'str' now for the entire column.